Food/kustomize.lua

```diff
@@ -0,0 +1,52 @@
+local name = "kustomize"
+local version = "3.8.2"
+
+food = {
+    name = name,
+    description = "Customization of kubernetes YAML configurations",
+    homepage = "https://kubernetes-sigs.github.io/kustomize/",
+    version = version,
+    packages = {
+        {
+            os = "darwin",
+            arch = "amd64",
+            url = "https://github.com/kubernetes-sigs/" .. name .. "/releases/download/" .. name .. "%2Fv" .. version .. "/" .. name .. "_v" .. version .. "_darwin_amd64.tar.gz",
```

Comment:
What’s with the %2F? Did you intend this to be a forward slash rather than the URL escape code?

Reply:
I was also slightly stumped by this, but it's the URL that comes out of https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv3.8.2

https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv3.8.2/kustomize_v3.8.2_linux_amd64.tar.gz

it does work in my terminal (linux, wsl2) so I guess it's ok like this? Open to suggestions.

Reply:
Oh I see why... The tag name is literally `kustomize/v3.8.2`. Odd tagging scheme. Regardless, it would seem this is the safer option.